wiki/Tournaments/GBC/GBC_2024_Spring/en.md

```diff
@@ -29,18 +29,36 @@ The **GB Cup 2024 Spring** (***GBC 2024 Spring***) is an osu!mania 4K point-base
 
 ## Prizes
 
+### QQ division
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "playoff stage 1st place") | Profile badge, KanonBot profile badge, 160 RMB |
+| ![Silver crown](/wiki/shared/crown-silver.png "playoff stage 2nd place") | KanonBot profile badge, 145 RMB |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "playoff stage 3rd place") | KanonBot profile badge, 130 RMB |
+| *4th place* | 115 RMB |
+| *5th place – 6th place* | 100 RMB |
+| *7th place – 8th place* | 85 RMB |
+| *9th place – 12th place* | 70 RMB |
+| *13th place – 16th place* | 55 RMB |
+| *17th place – 24th place* | 40 RMB |
+| *Full attendance prize* | KanonBot profile badge (all groups), 12 RMB (Group B), 18 RMB (Group A), 24 RMB (Group S), 30 RMB (Group EX) |
+
+![GBC 2024 Spring QQ division winner badge](img/GBC-2024-Spring-badge-QQ-1st.jpg) 
+
+### Discord division
+
 | Placing | Prize(s) |
 | :-: | :-- |
-| ![Gold crown](/wiki/shared/crown-gold.png "playoff stage 1st place") | Profile badge, KanonBot profile badge, *TBA* RMB |
-| ![Silver crown](/wiki/shared/crown-silver.png "playoff stage 2nd place") | KanonBot profile badge, *TBA* RMB |
-| ![Bronze crown](/wiki/shared/crown-bronze.png "playoff stage 3rd place") | KanonBot profile badge, *TBA* RMB |
-| *4th place* | *TBA* RMB |
-| *5th place – 6th place* | *TBA* RMB |
-| *7th place – 8th place* | *TBA* RMB |
-| *9th place – 12th place* | *TBA* RMB |
-| *13th place – 16th place* | *TBA* RMB |
-| *17th place – 24th place* | *TBA* RMB |
-| *Full attendance prize* | KanonBot profile badge (all groups), *TBA* RMB (Group B), *TBA* RMB (Group A), *TBA* RMB (Group S), *TBA* RMB (Group EX) |
+| ![Gold crown](/wiki/shared/crown-gold.png "playoff stage 1st place") | Profile badge, 6 months supporter |
+| ![Silver crown](/wiki/shared/crown-silver.png "playoff stage 2nd place") | 6 months supporter |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "playoff stage 3rd place") | 4 months supporter |
+| *4th place – 6th place* | $10 |
+| *7th place – 12th place* | 2 months supporter |
+| *13th place – 24th place* | $6 |
+| *Full attendance prize* | $2 (Group A), $3 (Group S), 1 month supporter (Group EX) |
```

Comment:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "playoff stage 1st place") | Profile badge, 6 months of osu!supporter |
| ![Silver crown](/wiki/shared/crown-silver.png "playoff stage 2nd place") | 6 months of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "playoff stage 3rd place") | 4 months of osu!supporter |
| *4th place – 6th place* | $10 |
| *7th place – 12th place* | 2 months of osu!supporter |
| *13th place – 24th place* | $6 |
| *Full attendance prize* | $2 (Group A), $3 (Group S), 1 month supporter (Group EX) |
```

Reply:
... is this correct? cash prizes between 4th-24th + attendance, but not for 7th-12th and 2nd-3rd?

seems a bit random so i want to confirm

Reply:
2 months of osu!supporter is equivalent to 8 USD and 1 month of osu!supporter is equivalent to 4 USD
im not sure if this is clear enough, asking the editor now

Reply:
adjusted the prize description to a specific amount

Reply:
> ... is this correct? cash prizes between 4th-24th + attendance, except 7th-12th and not for 2nd-3rd?
> 
> seems a bit random so i want to confirm

only full-attendance player will be able to participate in playoff, so the prize of playoff already include the full-attendance prize

Reply:
> ... is this correct? cash prizes between 4th-24th + attendance, except 7th-12th and not for 2nd-3rd?
> 
> seems a bit random so i want to confirm

only full-attendance player will be able to participate in playoff, so the prize of playoff already include the full-attendance prize